lib/wasix/src/state/func_env.rs

```diff
@@ -56,6 +60,15 @@ impl WasiFunctionEnv {
         let mut ctx = WasiFunctionEnv::new(&mut store, env);
         let (mut import_object, init) =
             import_object_for_all_wasi_versions(&module, &mut store, &ctx.env);
+
+        for ((namespace, name), value) in additional_imports {
+            // Note: We don't want to let downstream users override WASIX
+            // syscalls
+            if !import_object.exists(&namespace, &name) {
```

Comment:
Should this return an error instead of silently ignoring?
If not at least a `tracing::warn`.

Reply:
That's how it was implemented in `WasiEnv`, but I guess a trace message would make sense.